addons/project/views/rating_views.xml

```diff
@@ -6,14 +6,31 @@
         <field name="inherit_id" ref="rating.rating_rating_view_tree"/>
         <field name="mode">primary</field>
         <field name="arch" type="xml">
-            <field name="res_name" position="attributes">
-                <attribute name="string">Task</attribute>
+            <field name="rating_text" position="before">
+                <field name="create_date" position="move"/>
+                <field name="rated_partner_id" position="move"/>
+                <field name="partner_id" position="move"/>
+                <field name="parent_res_name" position="move"/>
+                <field name="res_name" position="move"/>
+                <field name="feedback" position="move"/>
+            </field>
+            <field name="rated_partner_id" position="attributes">
+                <attribute name="string">Assigned to</attribute>
+                <attribute name="optional">show</attribute>
+            </field>
+            <field name="partner_id" position="attributes">
+                <attribute name="optional">show</attribute>
             </field>
```

Comment:
The `optional` attribute is already set to `show` for this field in the parent view. 🙂 
See [here](https://github.com/odoo/odoo/blob/85b8a951167511852b0d1ecd21a36b0aaee474d1/addons/rating/views/rating_rating_views.xml#L11)

Reply:
Done.